packages/timelock-manager/deploy/1_TimelockManager.js

```diff
@@ -1,22 +1,87 @@
-module.exports = async ({ getNamedAccounts, deployments }) => {
-  const { deploy, log } = deployments;
-  const { deployer } = await getNamedAccounts();
-
-  const api3DaoAddresses = {
-    1: 0,
-    4: "0x0c26bb185ad09c5a41e8fd127bf7b8c99e81e5dc",
+const ethers = require("ethers");
```

Comment:
I guess this is for testing the frontend? Follow this to separate the contract deployment and population scripts (note the exported `tags` and how they are used in the npm scripts). 
https://github.com/osarrouy/api3-presale/tree/master/packages/contracts/deploy
We will be using this script to deploy on mainnet so would rather it not do anything else.